proto/cosmos/gov/v1/genesis.proto

```diff
@@ -17,10 +17,6 @@ message GenesisState {
   repeated Vote votes = 3;
   // proposals defines all the proposals present at genesis.
   repeated Proposal proposals = 4;
-  // params defines all the paramaters of related to deposit.
-  DepositParams deposit_params = 5;
-  // params defines all the paramaters of related to voting.
-  VotingParams voting_params = 6;
-  // params defines all the paramaters of related to tally.
-  TallyParams tally_params = 7;
+  // params defines all the paramaters of x/gov module.
+  Params params = 5;
```

Comment:
It seems to have cleaned up the code, but what will the impact to users be?

Reply:
I think some users will complain but it aligns with the rest of the repo that it makes it easier to deal with instead of creating custom logic for gov.

Reply:
This is a proto-breaking change, we should bump to gov v2 if we do this.

Why not just mark the exising fields as deprecated, and add comments that they will be ignored, to use `Params` instead

Reply:
> Why not just mark the exising fields as deprecated, and add comments that they will be ignored, to use Params instead

This is what I was going to suggest. In general, in such cases, when is it warranted to bump the proto package vs just keep fields and mark them as deprecated?